python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -374,71 +394,85 @@ async def _set_maximum_motion_values(self) -> None:
         vmax = self.config.amcs_vmax
         amax = self.config.amcs_amax
         jmax = self.config.amcs_jmax
-        self.log.info(f"Setting AMCS maximum velocity to {vmax}")
+        self.log.info(f"Setting AMCS maximum velocity to {vmax}.")
         vmax_dict: MaxValueConfigType = {"target": "vmax", "setting": [vmax]}
-        self.log.info(f"Setting AMCS maximum acceleration to {amax}")
+        self.log.info(f"Setting AMCS maximum acceleration to {amax}.")
         amax_dict: MaxValueConfigType = {"target": "amax", "setting": [amax]}
-        self.log.info(f"Setting AMCS maximum jerk to {jmax}")
+        self.log.info(f"Setting AMCS maximum jerk to {jmax}.")
         jmax_dict: MaxValueConfigType = {"target": "jmax", "setting": [jmax]}
         settings = [vmax_dict, amax_dict, jmax_dict]
         await self.config_llcs(system=LlcName.AMCS, settings=settings)
 
-    async def cancel_status_tasks(self) -> None:
-        """Cancel all status tasks."""
-        while self.status_tasks:
-            self.status_tasks.pop().cancel()
+    async def cancel_periodic_tasks(self) -> None:
+        """Cancel all periodic tasks."""
+        while self.periodic_tasks:
+            self.periodic_tasks.pop().cancel()
 
-    async def start_status_tasks(self) -> None:
-        """Start all status tasks."""
-        await self.cancel_status_tasks()
+    async def start_periodic_tasks(self) -> None:
+        """Start all periodic tasks."""
+        await self.cancel_periodic_tasks()
         methods_and_intervals = ALL_METHODS_AND_INTERVALS
         if self.simulation_mode == ValidSimulationMode.NORMAL_OPERATIONS:
-            methods_and_intervals = METHODS_AND_INTERVALS_FOR_COMMISSIONING
+            methods_and_intervals = {
+                methods_and_interval: execute
+                for (methods_and_interval, execute) in ALL_METHODS_AND_INTERVALS.items()
+                if ALL_METHODS_AND_INTERVALS[methods_and_interval]
+            }
         for method, interval in methods_and_intervals:
             func = getattr(self, method)
-            self.status_tasks.append(
-                asyncio.create_task(self.one_status_loop(func, interval))
+            self.periodic_tasks.append(
+                asyncio.create_task(self.one_periodic_task(func, interval))
             )
 
+    async def one_periodic_task(self, method: typing.Callable, interval: float) -> None:
+        """Run one method forever at the specified interval.
+
+        Parameters
+        ----------
+        method: coro
+            The periodic method to run.
+        interval: `float`
+            The interval (sec) at which to run the status method.
+
+        """
+        try:
+            while True:
+                await method()
+                await asyncio.sleep(interval)
+        except Exception:
+            self.log.exception(f"one_periodic_task({method}) failed.")
```

Comment:
Please state in the log message that you are continuing (despite the error), as otherwise I think readers of the message will expect that the task died at this point.

Also I gently suggest including repr(error) in the message (e.g. as `{e!r}`). That way it shows up in LOVE log message displays and simple Chronograf dashboards.

Reply:
The exception is caught outside the loop, so the task does not continue. I've made that clear and also added repr(error) in the statement as you suggested.

Reply:
Oops. Of course! No need to clarify anything; I should have seen that. Sorry!